companies/123greetings.json

```diff
@@ -1,22 +1,23 @@
 {
-    "slug": "123greetings",
+    "slug": "123greetings2",
```

Comment:
🚫 **[recordlint]** <[recordlint/some_error](https://github.com/datenanfragen/data/issues/1477)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>You made an error here.

abcdef
<details><summary>reviewdog suggestion error</summary>GitHub comment range and suggestion line range must be same. L2-L2 v.s. L0-L0</details>